Formula/liblo.rb

```diff
@@ -36,4 +35,21 @@ def install
 
     system "make", "install"
   end
+
+  test do
+    File.write(
```

Comment:
Please look how other formula do this. Generally we use an EOS block

Reply:
Thank you! Updated in 40b0806946735ca15adebaa02d87190c70353a2d